encoding/openapi/openapi.go

```diff
@@ -62,6 +62,9 @@ type Config struct {
 	// the empty string is returned.
 	DescriptionFunc func(v cue.Value) string
 
+	// AdditionalSchemasFunc allows adding additional schema constraints for a certain field.
```

Comment:
Rather than saying what it allows, I'd suggest saying how it works.
For example (using guessed-at semantics):

```
// AdditionalSchemasFunc is called for every definition in the spec when encoding CUE
// to OpenAPI. If it returns a non-empty slice, the constraints specified in the slice will
// be associated with that definition in the generated OpenAPI schema.
```

Also, out of interest, why not return a map (e.g. `map[string] ast.Expr`) rather than a slice. Does ordering matter?

Reply:
I think map is probably a good call, I will update this

Reply:
Done